### ISSUE TITLE
An example of regularization using Dropout  in the docs

### ISSUE THREAD

**Issue by Sentient07** [Theano/Theano#3923]
I felt similar to L1 and L2 regularization, we could also add an example of dropout technique.

**Comment by nouiz** [comment]
I think it would be great to document how this can be done directly in Theano. Many framework on top of Theano implement this, so you could check how they do it.

Do you want to do this?

**Comment by Sentient07** [comment]
@nouiz Yeah I'd gladly take this up. And I have already implemented a version of dropout over theano. Would add this by the end of the week. :+1:

**Comment by Sentient07** [comment]
@nouiz I guess this should be added at the end of http://deeplearning.net/tutorial/mlp.html#multilayer-perceptron this section. but i'm not able to locate where this file is. Could you please guide me on where to add this ?
cc : @lamblin

**Comment by lamblin** [comment]
The deep learning tutorials are in a separate repo, https://github.com/lisa-lab/DeepLearningTutorials
Rather than changing the `mlp` example, I think it would be better to add a new page and script about dropout.
The code would go into a `.py` file in https://github.com/lisa-lab/DeepLearningTutorials/tree/master/code, and the text (as RST) in a `.txt` file in https://github.com/lisa-lab/DeepLearningTutorials/tree/master/doc. You should use the `.. literalinclude::` directive in RST to include snippets from the code, rather than copy/pasting it.

**Comment by Sentient07** [comment]
@lamblin yeah, will fork that repo and send a PR there. Also, i had a look at the other files in that directory( for how documentations should look). I will try to write a good one. 
Thanks
